### ISSUE TITLE
better description when no exports found

### ISSUE THREAD

**Issue by marzelin** [webpack/webpack#4877]
** a *feature* **

**What is the current behavior?**

when `output.pathinfo` is set to `true` and no exports are found in a module, the module is labeled with: 
     `/* unknown exports provided */`

**What is the expected behavior?**
I find this message a bit misleading since it suggests that **there are some exports** but they are unknown for some reason. But as far as I know this message appears when webpack found no exports because **there are none** or it couldn't find any.

**If this is a feature request, what is motivation or use case for changing the behavior?**

I think that message like: `/* no exports found */` would be more appropriate.

**Please mention other relevant information such as the browser version, Node.js version, webpack version and Operating System.**

webpack@2.5.1

**Comment by sokra** [comment]
There could be CJS or AMD exports, but we can't detect them as CJS and AMD are dynamic and exports are only known at runtime.

**Comment by marzelin** [comment]
@sokra If the file has no exports (as it often is with the main file) the message it the same `/* unknown exports provided */`. When I first saw it I interpreted it as if there were some malicious exports I didn't know about, but it fact the module just didn't export anything. Current description is a bit misleading and I propose to replace it with `/* no static exports found */` to ease the pain for future webpack users that are not familiar with this practice.
